src/scenes/Level2Scene.js

```diff
@@ -0,0 +1,189 @@
+import "phaser";
+import Overlay from "../objects/Overlay";
+import TestObject from "../objects/characters/TestObject";
+import MapNavScene from "./MapNavScene";
+
+
+export default class Level2Scene extends Phaser.Scene {
+
+    static DEPTH_COMMAND;
+    static DEPTH_GROUND = 50;
+    static DEPTH_WALLS = 100;
+
+    controls;
+    map;
+    player;
+    sandBags;
+    ground;
+    exit;
+    speed = 100;
+    exitReached;
+    static LEVEL_NAME = 'LEVEL2';
+
+    constructor() {
+        super({key: 'Level2Scene'});
```

Comment:
Might be better to create a static variable (i.e. `Level2Scene.LEVEL_NAME`) and then pass it as the key. That way when the scene is started from the intro stage it can be done by calling `this.scene.start(Level2Scene.LEVEL_NAME, ...)`